File: codingTest/baekjoon/javascript/9093.js
let rl = require("readline").createInterface({ input: process.stdin, output: process.stdout });

let input = [];
rl.on("line", line => {
  input.push(line);
}).on("close", () => {
  let result = "";
  for (let i = 1; i < input.length; i++) {
    result +=
      input[i]
        .split(" ")
        .map(e => e.split("").reverse().join(""))
        .join(" ") + "\n";
  }
  console.log(result);
});

File: codingTest/baekjoon/python/9093.py
l = []
for _ in range(int(input())):
    l.append(input().split())

for item in l:
    for i, v in enumerate(item):
        item[i] = "".join(reversed(v))
    for i in item:
        print(i, end=" ")
    print()

# import sys
# input = sys.stdin.readline

# reverse__str=[]
# N=int(input())
# for i in range(N):
#     __str=input().split()
#     for arr in __str:
#         reverse__str.append(arr[::-1])
# print(' '.join(reverse__str))